views/awesome.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+
+{% block title%}{{title}} - Awesome{% endblock %}
+
+{% block bodyclass %}Awesome{% endblock %}
+
+{% block content %}
+<!-- awesome page test placeholder -->
+<div class="container">
+    <div class="inner_container">
```

Comment:
indentation is 2 spaces in all html files by default. This seems to have 3 (or 4?)